cli/index_test.ts

```diff
@@ -303,5 +303,53 @@ select 1 as \${dataform.projectConfig.vars.testVar2}
       },
       warehouseState: {}
     });
+
+    // Create a correctly formatted file
```

Comment:
One more thing actually. Could we extract formatting to a separate test? You can just copy paste the setup from the original test. It will be cleaner test will be more focused on formatting only.